packages/teams-js/test/public/media.spec.ts

```diff
@@ -940,10 +954,54 @@ describe('media', () => {
         return blobReadingPromise;
       }
 
-      it('getMedia calls with successful result via the handler', done => {
+      it('getMedia calls with successful result via the handler', async () => {
```

Comment:
This is a success test case, so the callback is being called. For the rest of the failure cases, we are checking with `done`(). I won't change the format just because it is a success case, I would want to keep it consistent for both success and failure cases.

Reply:
this is another piece of code that i just moved, but didn't change, I would be more than happy to discuss with you how you think these tests in this file should be structured.

Reply:
not sure what happened here but you're right, if we do it one place, we should do it everywhere and the code hadn't even been completely converted